lib/service/service.go

```diff
@@ -3280,13 +3280,14 @@ type proxyListeners struct {
 	ssh net.Listener
 	// sshGRPC receives gRPC traffic that is multiplexed on the Proxy SSH Port. When TLS routing
 	// is enabled only traffic with the TLS ALPN protocol common.ProtocolProxySSHGRPC is received.
-	sshGRPC       net.Listener
-	web           net.Listener
-	reverseTunnel net.Listener
-	kube          net.Listener
-	db            dbListeners
-	alpn          net.Listener
-	proxyPeer     net.Listener
+	sshGRPC           net.Listener
+	web               net.Listener
+	reverseTunnel     net.Listener
+	kube              net.Listener
+	db                dbListeners
+	alpn              net.Listener
+	reverseTunnelALPN net.Listener
```

Comment:
Can you add a short comment on why we add them and how is using them for the new listeners?